home/templates/home/tags/banners_list.html

```diff
@@ -0,0 +1,21 @@
+{% load static wagtailcore_tags wagtailimages_tags %}
+
+<div class="desktopBanners">
+    {% for banner in banners %}
+    <div class="bannerContainer">
+        <div class="desktopBannerContainer">
+            <div class="image">
+              {% if banner.specific.banner_link_page %}
+                <a href="{% pageurl banner.specific.banner_link_page %}">
+              {% elif banner.specific.external_link %}
+                <a href="{{ banner.specific.external_link }}" target="_blank">
+              {% else %}
+                <a href="#"></a>
```

Comment:
I think the anchor tag should not be closed here.

Reply:
removed unnecessary closing tag